scripting/pugsetup/natives.sp

```diff
@@ -501,7 +513,19 @@ public int Native_SetTeamBalancer(Handle plugin, int numParams) {
   bool override = GetNativeCell(2);
   if (!PugSetup_IsTeamBalancerAvaliable() || override) {
     g_BalancerFunctionPlugin = plugin;
+    //ArrayList players = new ArrayList();
     g_BalancerFunction = view_as<TeamBalancerFunction>(GetNativeFunction(1));
+    for (int i = 1; i <= MaxClients; i++) {
+      if (IsPlayer(i)) {
+        if (PugSetup_IsReady(i))
+          players.Push(i);
+      }
+    }
+    Call_StartFunction(g_BalancerFunctionPlugin, g_BalancerFunction);
+    Call_PushCell(players);
+    Call_Finish();
```

Comment:
Why is this changing? It seems incorrect to invoke this function since the native could be called far before the match is ready.